### DISCUSSION TITLE
Integration of maize-tech/laravel-markable

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#2495] [Help]
Hello,

I am using the package [maize-tech/laravel-markable](https://github.com/maize-tech/laravel-markable) which registers likes like this:
```php
<?php

namespace App\Models;

use App\Enums\PostTypeEnum;
use Illuminate\Database\Eloquent\Casts\Attribute;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\HasMany;
use Maize\Markable\Markable;
use Maize\Markable\Models\Like;


class Post extends Model
{
    use HasFactory;
    use Markable;

    protected $casts = [
        'id' => 'integer',
        'type' => PostTypeEnum::class,
    ];

    protected static array $marks = [
        Like::class,
    ];

    public function comments(): HasMany
    {
        return $this->hasMany(Comment::class);
    }

   public function user(): BelongsTo {
        return $this->belongsTo(User::class);
   }
}
```

Now I would like to integrate the like count but I am unsure how to do it in the table. Do you have any guidance how I may achieve this?
There is something like `Post::firstOrFail()->likers->count()` but I don't know how I can make it visible in the column.

**Comment by danharrin** [comment]
Is $model->likers a relation instance?

**Reply by Wulfheart** [reply]
Thanks for your reply. 
I think so.

**Reply by danharrin** [reply]
In which case `TextColumn::make('likes_count')->counts('likes')`, like you can for any other relationship?
